test/clustercoeffs_test.jl

```diff
@@ -26,4 +26,11 @@
         @test_throws ErrorException clustercoeffs(MatrixNetwork(-speye(4)))
         @test_throws ErrorException clustercoeffs(-speye(4))
     end
+    cc_mn = clustercoeffs(MatrixNetwork(A), false, false)
+    cc_csc = clustercoeffs(A, false, false)
+    cc_csr = clustercoeffs(sparse_to_csr(A)..., false, false)
+    cc_tri = clustercoeffs(findnz(A)[1], findnz(A)[2], false, false)
```

Comment:
Just out of curiosity, does the syntax...
`clustercoeffs(findnz(A)[1:2]..., false, false)`
work